src/main/java/de/zalando/tip/zalenium/servlet/live.java

```diff
@@ -0,0 +1,123 @@
+import org.openqa.grid.internal.utils.HtmlRenderer;
+import org.openqa.grid.web.servlet.RegistryBasedServlet;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+
+// We use this class name to be able to go to the resource like this: http://localhost:4444/grid/admin/live
+public class live extends RegistryBasedServlet {
+
+    public live(){
+        this(null);
+    }
+
+    public live(Registry registry) {
+        super(registry);
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        process(request, response);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        process(request, response);
+    }
+
+    protected void process(HttpServletRequest request, HttpServletResponse response)
+            throws IOException {
+
+        response.setContentType("text/html");
+        response.setCharacterEncoding("UTF-8");
+        response.setStatus(200);
+
+        StringBuilder builder = new StringBuilder();
+
+        builder.append("<html>");
+        builder.append("<head>");
+        builder.append("<script src='//ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js'></script>");
+        builder.append("<script src='/grid/resources/org/openqa/grid/images/console-beta.js'></script>");
+        builder.append("<link href='/grid/resources/org/openqa/grid/images/console-beta.css' rel='stylesheet' type='text/css' />");
+        builder.append("<link href='/grid/resources/org/openqa/grid/images/favicon.ico' rel='icon' type='image/x-icon' />");
+
+        builder.append("<title>Live Preview</title>");
+
+        builder.append("<style>");
+        builder.append(".busy {opacity : 0.4; filter: alpha(opacity=40);}");
+        builder.append("</style>");
+        builder.append("</head>");
+
+        builder.append("<body>");
+
+        builder.append("<div id='main_content'>");
+
+        builder.append(getHeader());
+
+        List<String> nodes = new ArrayList<>();
+        for (RemoteProxy proxy : getRegistry().getAllProxies()) {
+            if (proxy instanceof DockerSeleniumRemoteProxy) {
+                HtmlRenderer beta = new LiveNodeHtmlRenderer(proxy);
+                nodes.add(beta.renderSummary());
+            }
+        }
+
+        int size = nodes.size();
+        int rightColumnSize = size / 2;
+        int leftColumnSize = size - rightColumnSize;
+
+        builder.append("<div id='leftColumn'>");
+        for (int i = 0; i < leftColumnSize; i++) {
+            builder.append(nodes.get(i));
+        }
+        builder.append("</div>");
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Define a constant instead of duplicating this literal "</div>" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)